.deps/EXCLUDED/dev.md

```diff
@@ -1,8 +1,5 @@
 | Packages | Resolved CQs |
```

Comment:
If it does not break the script I would add some comment on why this file is needed. Like the following
```suggestion
This file contains a manual contribution to .deps/dev.md and it's needed because eclipse/dash-licenses does not deal with work-with CQs (more see https://github.com/eclipse/dash-licenses/issues/13)
| Packages | Resolved CQs |
```